### PR TITLE
Add travis step for building presto-admin artifacts

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

File: bin/build-artifacts-in-docker.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+CONTAINER_NAME="presto-admin-build-$(date '+%s')"
+CONTAINER_DIR="/mnt/presto-admin"
+
+docker run --name ${CONTAINER_NAME} -v ${ROOT_DIR}:${CONTAINER_DIR} --rm -i ${BASE_IMAGE_NAME}:${BASE_IMAGE_TAG} \
+  env CONTAINER_DIR="${CONTAINER_DIR}" bash <<"EOF"
+    cd ${CONTAINER_DIR}
+    pip install --upgrade pip==9.0.1
+    pip install -r requirements.txt
+    pip install tox-travis
+    export PYTHONPATH=${PYTHONPATH}:$(pwd)
```
Comment (sopel39) [inline bin/build-artifacts-in-docker.sh]: is this external `PYTHONPATH` valid for docker container?

Comment (losipiuk) [reply bin/build-artifacts-in-docker.sh]: this is internal PYTOHNPAT

File: .travis.yml [hunk 2]
```diff
@@ -32,7 +33,9 @@ before_script:
   - make presto-server-rpm.rpm
 script:
   - |
-    if [ -v SHORT_PRODUCT_TEST_GROUP ]; then
+    if [ -v ARTIFACTS ]; then
```
Comment (anusudarsan) [inline .travis.yml]: looks like this has to go in a separate if-else, otherwise it fails at line 56

Comment (losipiuk) [reply .travis.yml]: There was a type ARITFACTS vs ARTIFACTS. Just fixed that.

**Pull request merged**